storage/rocksdb/rdb_datadic.h

```diff
@@ -256,6 +256,27 @@ class Rdb_key_def {
     return i;
   }
 
+  /*
+    The same as get_first_key, but get the key for the last entry in the index
+  */
+  inline int get_last_key(uchar *const key, uint *const size) const {
+    if (m_is_reverse_cf)
+      get_infimum_key(key, size);
+    else
+      get_supremum_key(key, size);
+
+    /* Find out how many bytes of infimum are the same as m_index_number */
+    uchar unmodified_key[INDEX_NUMBER_SIZE];
+    rdb_netbuf_store_index(unmodified_key, m_index_number);
+    int i;
+    for (i = 0; i < INDEX_NUMBER_SIZE; i++) {
+      if (key[i] != unmodified_key[i])
+        break;
+    }
+    return i;
+  }
+
+
```

Comment:
clang-format is complaining about new line here.

Reply:
I'll fix these separately.